src/cocaine-app/infrastructure_cache.py

```diff
@@ -68,10 +97,10 @@ def _start_tq(self):
 
 class CacheItem(object):
 
-    def __init__(self, meta_session, idx_key, key_key, tq):
+    def __init__(self, meta_session, idx_key, key_tpl, task_queue):
         self.meta_session = meta_session.clone()
-        self.idx = indexes.SecondaryIndex(idx_key, key_key, self.meta_session)
-        self.__tq = tq
+        self.idx = indexes.SecondaryIndex(idx_key, key_tpl, self.meta_session)
+        self.__tq = task_queue
```

Comment:
It's better to do not include args renaming in this pull request.
Try to include in a pull request related and necessary diffs only next time.